File: z-clients/javascript/src/models/kvConfigureNamespaceIn.ts
// this file is @generated

export interface KvConfigureNamespaceIn {
    name: string;
}

export const KvConfigureNamespaceInSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): KvConfigureNamespaceIn {
        return {
            name: object['name'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: KvConfigureNamespaceIn): any {
        return {
            'name': self.name,
        };
    }
}

File: z-clients/rust/src/models/kv_configure_namespace_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct KvConfigureNamespaceIn {
    pub name: String,
}

impl KvConfigureNamespaceIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
